photon-client/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -24,12 +24,21 @@ import com.linkedin.photon.ml.avro.data.NameAndTermFeatureSetContainer
 import com.linkedin.photon.ml.data._
 import com.linkedin.photon.ml.evaluation.EvaluatorType._
 import com.linkedin.photon.ml.evaluation.{EvaluatorType, ShardedAUC, ShardedPrecisionAtK}
-import com.linkedin.photon.ml.normalization.NormalizationType
+import com.linkedin.photon.ml.normalization.{NormalizationContext, NormalizationType}
+import com.linkedin.photon.ml.stat.BasicStatisticalSummary
 import com.linkedin.photon.ml.test.SparkTestUtils
 import com.linkedin.photon.ml.util._
+import org.apache.spark.mllib.linalg.{Vector => MLVector}
+import org.apache.spark.rdd.RDD
+
+import com.linkedin.photon.ml.evaluation.Evaluator.EvaluationResults
+import com.linkedin.photon.ml.io.GLMSuite
+import com.linkedin.photon.ml.model.GAMEModel
 
 /**
  * Integration tests for GameEstimator.
+ *
+ * The test data set here is a subset of the Yahoo! music data set available on the internet.
```

Comment:
Either add a link or remove the internet part, I'm sure people will think to look there on their own